test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -49,68 +47,32 @@ public class RequestTests
             {
                 host.Start();
 
-                using (var client = new HttpClient())
+                using (var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
                 {
-                    var bytes = new byte[1024 * 1024];
-                    for (int i = 0; i < bytes.Length; i++)
-                    {
-                        bytes[i] = (byte)i;
```

Comment:
Hmm, this no longer checks for data corruption in the body... that's unfortunate.